packages/base64/package.json

```diff
@@ -33,7 +33,7 @@
   "scripts": {
     "build": "exit 0",
     "prepack": "tsc --build tsconfig.build.json",
-    "postpack": "git clean -f '*.d.ts*' '*.tsbuildinfo'",
+    "postpack": "git clean -fX \"*.d.ts*\" \"*.d.cts*\" \"*.d.mts*\" \"*.tsbuildinfo\"",
```

Comment:
I suspect that this will work:
```suggestion
    "postpack": "git clean -fX \"*.d.{,c,m}ts*\" \"*.tsbuildinfo\"",
```

Reply:
It does not

Reply:
(likewise this syntax is unsupported by `gitignore`)

Reply:
🐑,(bah)